### ISSUE TITLE
Load more than 5 company/education history

### ISSUE THREAD

**Issue by cmohrmueller** [tomquirk/linkedin-api#74]
I'm using this to construct a data set for my research, but unfortunately the experience and the education dictionary only contain the first 5 entries.
Do you plan to add a feature to load the rest sometime?

Thanks!

**Comment by plxel** [comment]
is there a way to request all of them?

**Comment by adnaanbheda** [comment]
Yeah, I'm looking for this too. The data seems to be inside a huge blob of text and I can't find any particular endpoints for education/experience either.

@tomquirk Any chances this can be done ?

**Comment by prastut** [comment]
+1 for the feature request 🥺

**Comment by tomquirk** [comment]
Ehhh. Just looked at this. Looks like the endpoint we're using only returns the first 5.

We have to rewrite the get_profile method to use the new linkedin api endpoint to get the entire list of experiences 😱 . This will take a while unforunately

**Comment by peteraust** [comment]
Hi there, is there any update on this topic or a workaround to get more than 5 entries?

**Comment by Prii-y** [comment]
> I'm using this to construct a data set for my research using my personal connections, but unfortunately the experience and the education dictionary only contain the first 5 entries. Do you plan to add a feature to load the rest sometime?
> 
> Thanks!

@cmohrmueller were u able to figure this out?

**Comment by fbuenohernandez** [comment]
@tomquirk How do you discover the API endpoints? I tried to digging into it but to no avail.

At least I discovered if you go into https://www.linkedin.com/in/user_link/details/experience/ you can see all the experiences.

I was trying to modify the code to reach this link but still no success.

**Comment by transitive-bullshit** [comment]
This is an example of the current API endpoint for the experience detail page: `https://www.linkedin.com/voyager/api/graphql?includeWebMetadata=true&variables=(profileUrn:urn%3Ali%3Afsd_profile%3AACoAAAdVCacB9uO3u3vDtvGPnDQeweefI2nV0gw,sectionType:experience)&queryId=voyagerIdentityDashProfileComponents.d946689b98e2b016a699a2e89028406f
`

**Comment by snisnik** [comment]
Anyone was able to make any progress on this?
